commit b74155483c09fa07d6a777a0ef880f6d82d15bb7
Author: Guillaume Ayoub <guillaume+gnome@courtbouillon.org>
Date:   Sat Jul 5 10:22:36 2025 +0000

    Fix OpenType features set for all petite caps
    
    Typo was introduced by b635e8e.

diff --git a/pango/pangofc-fontmap.c b/pango/pangofc-fontmap.c
index 47aced09..b27cf52a 100644
--- a/pango/pangofc-fontmap.c
+++ b/pango/pangofc-fontmap.c
@@ -2198,7 +2198,7 @@ pango_fc_make_pattern (const  PangoFontDescription *description,
       break;
     case PANGO_VARIANT_ALL_PETITE_CAPS:
       maybe_add_feature (pattern, features, "pcap=1");
-      maybe_add_feature (pattern, features, "c2sc=1");
+      maybe_add_feature (pattern, features, "c2pc=1");
       break;
     case PANGO_VARIANT_UNICASE:
       maybe_add_feature (pattern, features, "unic=1");
